### PR TITLE
Remove warmup epochs from training options

### PR DESCRIPTION
**Pull request by frostedoyster**
Fixes the `pet-finetuning` example

### CODE REVIEW

**Comment by pfebrer** [comment]
Nice thanks, so the effect of removing the input is the same as the previous `num_epochs_warmup: 1`?

**Comment by pfebrer** [comment]
Should we update the `metatrain` dependency to only the versions with the new scheduler?
